lib/event_sourcery/event_body_serializer.rb

```diff
@@ -9,16 +14,25 @@ def initialize
       @serializers = Hash.new(IdentitySerializer)
     end
 
+    # Register a serializer (instance or block) for the specified type
+    #
+    # @param type The type for which the provided serializer will be used for
+    # @param serializer Optional. A serializer instance for the given type
+    # @param block_serializer [Proc] Optional. A block that performs the serialization
     def add(type, serializer = nil, &block_serializer)
       @serializers[type] = serializer || block_serializer
       self
     end
 
+    # Serialize the given event body
+    #
+    # @param event_body event body to be serialized
     def serialize(object)
       serializer = @serializers[object.class]
       serializer.call(object, &method(:serialize))
     end
 
+    # Built in implementation for serializing Hash objects
```

Comment:
Maybe "Default serializer for Hash objects" rather than "Built in implementation"?
Every implementation here is "built-in", maybe the fact that this implementation is the default one should be emphasised?